README.md

```diff
@@ -7,8 +7,6 @@
 - CSS
 - Javascript
 
-
-
```

Comment:
- [ ] Please add all collaborator names and social links in the Authors section on your README file because now I can see only name and social links for one collaborator.

Reply:
added my social @Omar-Muhamad